client/signup/steps/survey/survey-step-v1.jsx

```diff
@@ -98,25 +98,47 @@ export default React.createClass( {
 					</div>
 
 					<Card className={ subVerticalsClasses }>
-						<BackButton isCompact className="survey-step__title" onClick={ this.showStepOne }>{ this.state.stepOne && this.state.stepOne.label }</BackButton>
+						<BackButton isCompact className="survey-step__title" onClick={ this.showStepOne }>
+							{ this.state.stepOne && this.state.stepOne.label }
+						</BackButton>
 						{ this.state.stepTwo.map( this.renderStepTwoVertical ) }
 					</Card>
 				</div>
 			</div>
 		);
 	},
 
+	renderHeaderText() {
+		if ( this.props.surveySiteType === 'blog' ) {
+			const { domain } = this.props.queryObject;
+
+			if ( domain ) {
+				return this.translate( 'Start blogging on %(domain)s today!', {
+					args: { domain }
+				} );
+			}
+
+			return this.translate( 'Create your blog today!' );
+		}
+
+		return this.translate( 'Create your site today!' );
+	},
+
+	renderSubHeaderText() {
+		return this.props.surveySiteType === 'blog'
+			? this.translate( 'WordPress.com is the best place for your WordPress blog.' )
```

Comment:
Since you removed the check on the flow name this will be presented in all other flows. Is this something we want?